bin/nodetool

```diff
@@ -98,6 +71,8 @@ if [ "x$MAX_HEAP_SIZE" = "x" ]; then
     MAX_HEAP_SIZE="128m"
 fi
 
+validate_env JMX_PORT
```

Comment:
could you please remind me why we are checking just JMX_PORT here? Seems to be quite random but I guess it has some logic in it?